Lib/pydoc.py

```diff
@@ -2074,19 +2074,21 @@ def help(self, request, is_cli=False):
 
     def intro(self):
         self.output.write('''
-Welcome to Python {0}'s help utility!
-
-If this is your first time using Python, you should definitely check out
-the tutorial on the internet at https://docs.python.org/{0}/tutorial/.
+Welcome to Python {0}'s help utility! If this is your first time using
+Python, you should definitely check out the tutorial on the internet at
```

Comment:
"on the internet at [URL]" is somewhat redundant. 

For brevity:
```suggestion
Python, you should definitely check out the tutorial at
```

Reply:
Kind of quaint these days ;-).